learning/tour-of-beam/frontend/lib/pages/tour/controllers/unit.dart

```diff
@@ -18,28 +18,40 @@
 
 import 'package:flutter/widgets.dart';
 import 'package:get_it/get_it.dart';
+import 'package:playground_components/playground_components.dart';
 
 import '../../../cache/unit_progress.dart';
+import '../../../models/event_context.dart';
+import '../../../models/unit.dart';
 import '../../../repositories/client/client.dart';
+import '../../../services/analytics/events/unit_completed.dart';
 
 class UnitController extends ChangeNotifier {
-  final String unitId;
-  final String sdkId;
+  final UnitModel unit;
```

Comment:
What is a UnitModel?  Is this the model class representing the unit?  Should this be documented here as a code comment?  Also to improve code readability, could we prefix all classes coming from outside with the library it is imported from?  For example, I would have to open each imported file here to find where UnitModel comes from or do a search for its symbol.

Reply:
Added a comment to the two classes instead.
Regarding the library prefixes I answered above.